studyzen/Flashcards/FlashcardsController.cs

```diff
@@ -50,6 +50,6 @@ public IActionResult UpdateFlashcardById(int flashcardId, [FromBody] UpdateFlash
     public IActionResult DeleteFlashcard(int flashcardId)
     {
         _flashcardService.DeleteFlashcard(flashcardId);
-        return NoContent();
+        return Ok();
```

Comment:
If Hamlet was a coder, he would ask, "Empty response body, or no response body, that is the question"